rholang/reference_doc/new/README.md

```diff
@@ -1,31 +1,27 @@
-## 'new'
+## New construct
 
 ### Syntax
-- `new NameDeclarations in { Process[Names Declared] }`
-
-where `NameDeclarations` are a nonempty, comma separated list with terms of the form
-- `NameVariable`
-- `NameVariable( Uri )`
-
-Examples:
-- `new x,y,z in { Process[x,y,z] }`
-
-Nested is equivalent to listed, i.e.
-
- new x,y,z in { Process[x,y,z] }
-
-is equivalent to
-
-  new x in {
-   new y in {
-    new z in {
-      Process[x,y,z]
-     }
-    }
-   }
-
-s the free variables in `Process` and substitutes them with unforgeable names (see unforgeable names section).
-
-This is to make a private channel. Makes concurrent computation without any issues of shared memory possible.
-
-Sending on a public channel is pretty much never useful. So, no sends/listens on a public (forgeable) channel are committed to the tuple space.
+```
+    New ::= "new" NameDecs "in" "{" Proc "}"
+    NameDec  ::= NameVar | NameVar "(" Uri ")"
+    NameDecs ::= NameDec | NameDec "," NameDecs
+```
+where `NameVar` is a `Name` variable and `Uri` a uri in backticks.
+
+The `new` construct serves as a variable binder with scope `Proc` which produces an unforgeable process for each uniquely declared variable and substitutes these processes for the variables. Since these are technically name variables, the substitution mapping is between the declared variables and their corresponding *quoted* unforgeable process.
+
+New should be flattenable & name declarations set-like (monadic without unit?):
```

Comment:
What do you mean by `monadic without unit?`?

Reply:
I never defined it... I mean that `new x in {new y in {P}} == new x,y in {P}`, but this, in fact, false! I'll delete this comment.